### DISCUSSION TITLE
What is this weird hidden sub-domain i found?

### DISCUSSION THREAD

**Discussion by polygoners1** [PenguinMod/PenguinMod-Home#116] [Q&A] [answered]
so i was looking through domains under penguinmod.com and i found [this!](https://penguinmod.com/05-02-1998_test_screenshot.png) i have no clue why this is here. it has existed since at least november 10th 2023. pls help.

**Comment by RedMan13** [comment, marked as answer]
its a screen shot of 3d being tested
I think this from when Jeremy was working on penguinmod vr

**Reply by polygoners1** [reply]
thanks!

**Reply by JeremyGamer13** [reply]
this is actually not a screenshot of either of those :trollface:

**Reply by RedMan13** [reply]
oh damn so its just fake? :trollface:

**Reply by polygoners1** [reply]
no, its real. here is the proof
![Screenshot 2024-07-15 9 09 41 AM](https://github.com/user-attachments/assets/71ae3d32-c114-4ee2-a4c8-a97c21482027)

**Reply by MerrCraft** [reply]
> this is actually not a screenshot of either of those :trollface:

so then what is it?
